drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/widgets/ScenarioGrid.java

```diff
@@ -73,6 +73,8 @@ public void setContent(Simulation simulation) {
         ((ScenarioGridModel) model).bindContent(simulation);
         setHeaderColumns(simulation);
         appendRows(simulation);
+        ((ScenarioGridModel) model).loadFactMappingsWidth();
```

Comment:
@yesamer 
Please remove those two lines and set the column' width directly at column creation inside `ScenarioGrid.setHeaderColumn` (I think it should be possible))

Reply:
@gitgabrio As explained, this is required because, during the insertion of the column the grid, the current logic re-calculate the column dimension based on the number of the columns present in the grid.